src/index.html

```diff
@@ -2,16 +2,267 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
     <script type="text/javascript" src="./main.js"></script>
+    <header class="header"></header>
+
+    <main class="main">
+      <div class="content">
+        <form
+        action="http://localhost:8080/api" method="post"
+        name="taskForm"
+        >
+          <fieldset class="group personalInfo">
+            <legend class="legend personalInfo__legend">
+              Personal information
+            </legend>
+            <div class="content personalInfo__content">
```

Comment:
Delete this div. You added a lot of redundant classes, but they don do nithing. Please keep only two classes: 
1. for fieldset with margin between groups
2. for form-field with margin between inputs